car_service/request_repo.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'request'
+
+class Repo
+
+  def initialize
+    @requests = []
+  end
+
+  #add a new request
```

Comment:
I don't think those comments are necessary. The code is self-explanatory, in my opinion.